app/src/main/java/com/concordium/wallet/ui/passphrase/recover/PassPhraseRecoverViewModel.kt

```diff
@@ -76,10 +80,17 @@ class PassPhraseRecoverViewModel(application: Application) : AndroidViewModel(ap
         _validateLiveData.value = success
     }
 
-    fun setSeedPhrase(seed: String, password: String) = viewModelScope.launch {
+    fun setSeedPhrase(seedPhrase: String, password: String) = viewModelScope.launch {
```

Comment:
This is almost a duplicate of the other `setSeedPhrase`, and the same goes for `saveSeedPhrase`. It would be better if we could remove the duplication, but otherwise the comments I had for the other one goes for this one as well.